test.disabled/ejabberd_tests/tests/mod_global_distrib_SUITE.erl

```diff
@@ -242,6 +248,32 @@ generic_end_per_testcase(CaseName, Config) ->
 %% Service discovery test
 %%--------------------------------------------------------------------
 
+test_host_refreshing(_Config) ->
+    timer:sleep(?HOSTS_REFRESH_INTERVAL + 20), % Let's wait to be sure refresher added children
+    Children = rpc(asia_node, supervisor, which_children, [mod_global_distrib_outgoing_conns_sup]),
+    ct:log("children in asia: ~p~n", [Children]),
+    ct:log("redis: ~p~n", [rpc(asia_node, mod_global_distrib_mapping, hosts, [])]),
+    ct:log("children in europe1: ~p~n", [rpc(europe_node1, supervisor, which_children, [mod_global_distrib_outgoing_conns_sup])]),
+    [EuropeHost] = lists:filtermap(fun({europe_node1, Host, _}) -> {true, list_to_binary(Host)};
+                                      (_) -> false end, get_hosts()),
+    EuropeSup = binary_to_atom(<<"mod_global_distrib_server_sup_", EuropeHost/binary>>, utf8),
+    [EuropePid] = lists:filtermap(fun({Sup, Pid, supervisor, _}) -> {Sup =:= EuropeSup, Pid};
+                                   (_) -> false end,
+                                      Children),
+
+    erlang:exit(EuropePid, kill),
+
```

Comment:
Add timer:sleep(5000) here and run this test.
It's how travis would execute this test. Sometimes.

Correct order

```
ChildrenNoEurope = rpc(asia_node, supervisor, which_children, [mod_global_distrib_outgoing_conns_sup]),
erlang:exit(EuropePid, kill),
```